widgy/contrib/form_builder/models.py

```diff
@@ -444,7 +444,12 @@ def friendly_uuid(uuid):
     """
     result = base64.b32encode(hashlib.sha1(force_bytes(uuid)).digest()[:5]).lower()
     # avoid accidental profanity
-    return result.replace(b'e', b'0').replace(b'o', b'1').replace(b'u', b'8').replace(b'i', b'9')
+    profanity_filter = (result.replace(b'e', b'0')
+                              .replace(b'o', b'1')
+                              .replace(b'u', b'8')
+                              .replace(b'i', b'9'))
+    # Python 2/3 compatibility
+    return bytes.decode(profanity_filter, 'ascii')
```

Comment:
The normal way to call methods is by retrieving them from the instance, not the class. You could also clean up the line noise by converting to unicode before doing the replacing.